flint.ui/src/views/gcbm/GCBMUpload.vue

```diff
@@ -4,7 +4,7 @@
       <div
         class="relative bg-gradient-to-tl from-green-300 via-blue-500 to-purple-600 md:pt-32 pb-32 pt-12 w-full h-auto"
       >
-        <div class="px-4 md:px-10 mx-auto w-full">
+        <div class="  mx-auto w-full">
```

Comment:
![image](https://user-images.githubusercontent.com/35102959/159175731-22d14a69-18b8-4606-9205-9bc04af7077d.png)
there's a warning in the terminal too. can you please fix all the extra spaces in the files you edited?